Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -925,6 +926,52 @@ void HoI4::World::reportIndustryLevels() const
 }
 
 
+void HoI4::World::setTrainMultipliers()
+{
+	auto greatPower = greatPowers.begin();
+	if (greatPower == greatPowers.end())
+	{
+		return;
+	}
+	(*greatPower)->setTrainsMultiplier(5.0F);
+
+	++greatPower;
+	if (greatPower == greatPowers.end())
+	{
+		return;
+	}
+	(*greatPower)->setTrainsMultiplier(4.0F);
+
+	++greatPower;
+	if (greatPower == greatPowers.end())
+	{
+		return;
+	}
+	(*greatPower)->setTrainsMultiplier(3.0F);
+
+	++greatPower;
+	if (greatPower == greatPowers.end())
+	{
+		return;
+	}
+	(*greatPower)->setTrainsMultiplier(2.0F);
+
+	++greatPower;
+	if (greatPower == greatPowers.end())
+	{
+		return;
+	}
+	(*greatPower)->setTrainsMultiplier(2.0F);
```

Comment:
Same multiplier as above, intended?

Reply:
Yep, more explicit with the simpler loop now.